BUILD.bazel

```diff
@@ -46,6 +49,55 @@ container_bundle(
     },
 )
 
+container_bundle(
+    name = "e2e-env-docker-loadable",
+    images = {
+        "{STABLE_IMG_REGISTRY}/{STABLE_IMG_REPOSITORY}/cip-e2e-env:latest": "//:e2e-env",
+        "{STABLE_IMG_REGISTRY}/{STABLE_IMG_REPOSITORY}/cip-e2e-env:{IMG_TAG}": "//:e2e-env",
+    },
+)
+
+toolchain_container(
```

Comment:
Genuinely haven't seen this before.  Why are we using a toolchain_container as opposed to a "normal" container rule?